uniswap_swap_bot/src/bot.spec.ts

```diff
@@ -0,0 +1,78 @@
+import {
+  FindingType,
+  FindingSeverity,
+  Finding,
+  HandleTransaction,
+  createTransactionEvent,
+} from "forta-agent";
+import { createAddress } from "forta-agent-tools";
+
+import bot from "./bot";
+import { UNISWAP_ADDRESS, SWAP_EVENT } from "./constant";
+
+describe("swap occur", () => {
+  let handleTransaction: HandleTransaction;
+
+  const mockTxEvent = createTransactionEvent({} as any);
+
+  beforeAll(() => {
+    handleTransaction = bot.handleTransaction;
+  });
+
+  describe("token swap", () => {
+    const sender = createAddress("0x1");
+    const recipient = createAddress("0x2");
+    const amount0 = 300;
+    const amount1 = 100;
+    const sqrtPricex96 = 2;
+    const liquidity = 5;
+    const tick = 10;
+    it("returns empty findingd if there is no swap", async () => {
```

Comment:
```suggestion
    const tick = 10;
    
    it("returns empty findingd if there is no swap", async () => {
```
Adding spacing in between the variables and `it` test block for legibility.